src/config/systemConfigs/systemConfigs.js

```diff
@@ -941,255 +941,15 @@ module.exports = {
-                host: '54.213.89.75',
-                port: 26656,
-              },
-              {
-                id: 'ecaf7271938a3b8efbdb6bc81252e2c42a6570f0',
-                host: '52.32.232.156',
-                port: 26656,
-              },
-              {
-                id: '9d6291f557433f21ed20b5a2ec5f7fc8bf276a47',
-                host: '34.212.226.44',
-                port: 26656,
-              },
-              {
-                id: '2d5b6c6ce4323916f4f7d04e083c60ed0eea5780',
-                host: '35.160.140.37',
-                port: 26656,
-              },
-              {
-                id: 'f6721a9f4968ecf0617172c31e93511c71da3c78',
-                host: '52.11.133.244',
-                port: 26656,
-              },
-              {
-                id: '5c67a79c3b0ae3a84d78957c5464e5a43ece2a50',
-                host: '54.201.189.185',
-                port: 26656,
-              },
-              {
-                id: '186a6370ee79e4fcc5e212dde1ef3f44116d3335',
-                host: '34.216.200.22',
-                port: 26656,
-              },
-              {
-                id: 'eb611c99d91cc7c01eb2447879bcda747f533ba6',
-                host: '34.220.144.226',
-                port: 26656,
-              },
-              {
-                id: '3f8044db1e624b837246a530c94e5f7908ef44ca',
-                host: '34.216.195.19',
-                port: 26656,
-              },
-              {
-                id: '71bf8f7fc2547732426560d9608bc98ff548206e',
-                host: '54.185.1.69',
-                port: 26656,
-              },
-              {
-                id: '211eb0d62eeb899a3fd1075f7bd4c5077153cba2',
-                host: '35.166.29.155',
-                port: 26656,
-              },
-              {
-                id: 'c23c5da6ff8f2a95c2e25e64bde421b6a05dc1fc',
-                host: '18.237.147.160',
-                port: 26656,
-              },
-              {
-                id: 'a747ff41e78b89fc46999d5b8cf8874059cb74f7',
-                host: '34.220.228.214',
-                port: 26656,
-              },
-              {
-                id: '76fc3cf8d9e8ce937afa0734535b2cfcda1eb3c3',
-                host: '54.212.75.8',
-                port: 26656,
-              },
-              {
-                id: '94364e1b8336aaab5b0adbfc813fb3747ffe9894',
-                host: '35.163.152.74',
-                port: 26656,
-              },
-              {
-                id: '39045ca8ef8b560387b0612e02cd591c3d72e9e9',
-                host: '18.236.78.191',
-                port: 26656,
-              },
-              {
-                id: '7be31054b6b3687c27e9c618912b4bcb114201ef',
-                host: '54.148.106.179',
-                port: 26656,
-              },
-              {
-                id: '6bdaf339bf7ec90869162934a7145dde93d077c3',
-                host: '34.211.46.222',
-                port: 26656,
-              },
+            seeds: [
```

Comment:
You also need to add `seeds: [],` on line 92 or other configs won't validate

Reply:
Thanks. Fixed.